ups/ci_hsc.table

```diff
@@ -4,7 +4,7 @@ setupRequired(meas_extensions_psfex)
 setupRequired(meas_extensions_shapeHSM)
 setupRequired(numpy)
 setupRequired(obs_subaru)
-setupRequired(pex_logging)
+setupRequired(log)
```

Comment:
Why not keep this alphabetically sorted? It's not a big deal, but it seems a shame to break it.